docs/advanced-features/measuring-performance.md

```diff
@@ -1,15 +1,15 @@
 ---
-description: Measure and track page performance using Next.js Analytics
+description: Measure and track page performance using Next.js Speed Insights
 ---
 
 # Measuring performance
 
-[Next.js Analytics](https://nextjs.org/analytics) allows you to analyze and measure the performance of
+[Next.js Speed Insights](https://nextjs.org/analytics) allows you to analyze and measure the performance of
```

Comment:
Do we want to rename (and redirect) https://nextjs.org/analytics to https://nextjs.org/speed-insights too?

Reply:
Not yet. This will likely happen in the future